src/utils/shell_load_directory.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+#  make sure DB_USER, DB_PASS and DB_URL are properly set as well
+#  as BINDIR. then
+#
+#    python shell_load_directoy.py  JSONdirectory
```

Comment:
do you mean `bash shell_load_directory.sh <dir>`?

Reply:
Yep - just pushed the fix